lib/leaf/terms.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function( property, value, parameters ){
```

Comment:
here it's called `parameters` while in the others it's called `extra`, is that intentional?

Reply:
Yeah, i meant to rename them both to `parameters`, which is a better name. This is now fixed.